monit/default.toml

```diff
@@ -0,0 +1,19 @@
+check_interval = 30
+log = 'syslog'
+alert_recipients = []
+
+[eventqueue]
+    enabled = false
```

Comment:
Remove the indentation for the toml

Reply:
Argh, there seems to be some bikesheding to do here ;) I'm on it

Reply:
To make you feel better, I definitely hate myself for making this comment 😬